src/uniprotkb/components/results/styles/group-by.module.scss

```diff
@@ -0,0 +1,136 @@
+@import 'franklin-sites/src/styles/colours';
+
+.groupby-container {
+  .autocomplete {
+    width: 50ch;
+  }
+
+  .no-results {
+    max-width: 30rem;
+    margin: 1rem;
+  }
+
+  ul.groupby {
+    border-left: 1px solid $colour-gainsborough;
+    margin-left: 1rem;
+    margin-bottom: 0;
+
+    &__header {
+      border-left: 0;
+      margin-left: 0;
+      border-bottom: 1px solid $colour-gainsborough;
+      padding-bottom: 0.5rem;
+
+      li.header {
+        h3 {
+          display: inline-block;
+        }
+        .count {
+          display: inline-block;
+          width: 17ch;
+        }
+      }
+
+      li.parent {
+        .count {
+          display: inline-block;
+          width: 15ch;
+        }
+        .active-label {
+          margin-right: 0.5rem;
+        }
+      }
+    }
+
+    // The first ul to follow the header which may be either "ancestors" or "groups" from the API
+    &__header + ul {
+      border-left: 0;
+      margin-left: -1ch;
+      margin-top: 0.5rem;
+      margin-bottom: 2rem;
+
+      .count {
+        display: inline-block;
+        width: 9.5ch;
+      }
+    }
+
+    .warning {
+      margin-left: 0.75rem;
+      color: $colour-warning;
+    }
+
+    .spinner {
+      margin-left: -0.25rem;
+    }
+
+    li.node {
+      white-space: nowrap;
+
+      // "highlight" currently hovered or focused row
+      &:hover,
+      &:focus-within {
+        & > .label > a {
+          position: relative;
+          &::after {
+            z-index: -1;
+            content: '';
+            position: absolute;
+            top: -0.25ch;
+            bottom: -0.25ch;
+            left: -0.75ch;
+            right: -0.75ch;
+            opacity: 0.4;
+            background-color: $colour-warning;
+          }
+        }
+      }
+
+      .expand {
+        width: 4ch;
+        display: inline-block;
+        height: 1ch;
+
+        button {
+          padding: 0;
+          border: 0;
+          margin-top: -0.1rem;
+          margin-bottom: 0rem;
+          margin-left: 0.725rem;
+          transition: transform 0.25s ease-out;
+          outline: 0;
```

Comment:
```suggestion
```

Would suggest removing this, as when tabbing around we can't see where the focus is